app/scripts/components/app/header.js

```diff
@@ -22,12 +22,18 @@ const paths = [
 var Header = React.createClass({
   displayName: 'Header',
   propTypes: {
-    location: React.PropTypes.object,
-    dispatch: React.PropTypes.func,
     api: React.PropTypes.object,
+    apiVersion: React.PropTypes.object,
+    dispatch: React.PropTypes.func,
+    location: React.PropTypes.object,
     minimal: React.PropTypes.bool
   },
 
+  componentWillMount: function () {
+    const { dispatch, api } = this.props;
+    if (api.authenticated) dispatch(getApiVersion(compatibleApiVersions));
```

Comment:
Is there a reason to pass the `compatibleApiVersions` to the action rather than loading the variable directly in the actions file? It would just require less wiring between the component and the action

Reply:
Good call. I've updated the code.